cmd/ftl/cmd_deploy.go

```diff
@@ -53,6 +54,37 @@ func (d *deployCmd) Run(ctx context.Context, client ftlv1connect.ControllerServi
 		return errors.WithStack(err)
 	}
 
+	module := schemapb.Module{
+		Name: config.Module,
+	}
+	// TODO(worstell): clean up deploy cmd to handle language-specific stuff
+	if config.Language == "kotlin" {
```

Comment:
This will be used for Go as well, so we can remove this condition.

Reply:
sg - i was thinking we'd eventually merge `ftl-go` behavior into `ftl deploy` and remove this during the refactor because i think presently this logic would break for Go modules (but maybe `ftl deploy` is already broken for go?)

will remove it!

Reply:
Yeah ftl delpoy doesn't currently work with Go